File: tests/fixtures/fixture_java/src/main/java/com/app/services/OrderService.java
package com.app.services;

import com.app.models.Order;

public class OrderService {
    public Order placeOrder(double[] items) {
        double total = Order.calculateTotal(items);
        return new Order("o1", "u1", total);
    }

    public boolean cancelOrder(String orderId) {
        return orderId != null;
    }
}

File: tests/fixtures/fixture_csharp/Services/OrderService.cs
using App.Models;

namespace App.Services
{
    public class OrderService
    {
        public Order PlaceOrder(decimal[] items)
        {
            var total = Order.CalculateTotal(items);
            return new Order { Id = "o1", UserId = "u1", Total = total };
        }
    }
}
